### ISSUE TITLE
Remember position on timeline when switching accounts

### ISSUE THREAD

**Issue by jbalint-de** [mastodon/mastodon-android#471]
When switching accounts the timeline is always shown at the latest post. Instead the app should remember timeline positions when switching between multiple accouts. 

Somewhat similar to #70, #147 and #150, but no duplicate.

**Comment by realpixelcode** [comment]
Yes, exactly that! It's always frustrating not being able to return to the last position in the timeline, regardless of the situation or the reason.

**Comment by C-Duv** [comment]
Looks similar to an issue on the iOS application (https://github.com/mastodon/mastodon-ios/issues/744) when switching tab.

**Comment by strk** [comment]
Isn't a timeline strictly tight to a given instance ? I have 6 accounts on 6 different instances so I'm not sure I could access the same timeline (on the same spot) when switching accounts.

But I often feel the need to open a given post as another account. This is a feature Tusky has: a menu item to open a given post as someone else. Could this also solve your needs ? I wanted to file a feature request but maybe we could re-use this one ?

**Comment by realpixelcode** [comment]
It's not about remembering the timeline position when switching to another profile (that's not possible, as you correctly said).

Instead, it's about remembering the timeline position when **_coming back_** after switching profiles.

1. Scroll down in the timeline of profile A.
2. Switch to profile B.
3. Come back to A.
4. Mastodon has not remembered the timeline position from step 1.
